application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -102,6 +121,31 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
             .insert());
     }
 
+    private void handleListCommand(SlashCommandInteractionEvent event) {
+        BiFunction<Long, Instant, String> getDescription = (channelId, remindAt) -> """
+                Channel: %s
+                Remind at: %s""".formatted(MessageUtils.mentionChannelById(channelId),
+                TimeFormat.DEFAULT.format(remindAt));
+
+        EmbedBuilder remindersEmbed = new EmbedBuilder().setTitle("Pending reminders")
+            .setColor(RemindRoutine.AMBIENT_COLOR);
+
+        database
+            .read(context -> context.selectFrom(PENDING_REMINDERS)
+                .where(PENDING_REMINDERS.GUILD_ID.eq(event.getGuild().getIdLong())
+                    .and(PENDING_REMINDERS.AUTHOR_ID.eq(event.getUser().getIdLong())))
+                .orderBy(PENDING_REMINDERS.CREATED_AT.asc())
+                .fetch())
+            .forEach(reminder -> remindersEmbed.addField(reminder.getContent(),
+                    getDescription.apply(reminder.getChannelId(), reminder.getRemindAt()), false));
```

Comment:
Here again i would be using multiple variables to store the field values. You dont have to put everything in one line. Use some curly brackets and add it in a readable way please.